doc/ale-zig.txt

```diff
@@ -7,6 +7,14 @@ Integration Information
 
   Currently, the only supported linter for zig is zls.
 
+g:ale_zig_zig_executable                             *g:ale_zig_zig_executable*
```

Comment:
For consistency better call these ziffmt..  If zig supports other tools in the future we can differentiate them if we use different name than the executable:

```
g:ale_zig_zigfmt_executable                                                 *g:ale_zig_zigfmt_executable*
```

Reply:
Done